CHANGELOG.md

```diff
@@ -4,28 +4,28 @@ All notable changes to this project will be documented in this file.
 
 ## [x.x.x] - UNRELEASED
 
-### Overview
-
-### EDC UI
-
-### EDC-Extensions
-
-#### Major Changes
-
-#### Minor Changes
+### Major Changes
 
-#### Patch Changes
+- Upgrade to core-EDC version `0.1.2`
+- Now using the `Dataspace Protocol`
+- Major changes to the management API
+  - Examples for the new requests are located in the postman collection in the `docs` folder
+  - The `OpenAPI` file has been updated to represent the EDC version `0.1.2`
 
-### Deployment Migration Notes
+### Patch Changes
 
-#### Compatible Versions
+- Fix API Wrapper Broker Server API Spec not showing in client libraries.
```

Comment:
Should be not showing in client libraries as it has been removed entirely from edc-extensions. Is this a rebasing artifact?

Reply:
Might origin from rebasing, I deleted this change